src/wrappers/python/PyOpenEXR.cpp

```diff
@@ -347,114 +352,97 @@ PyPart::readPixels(MultiPartInputFile& infile, const ChannelList& channel_list,
     part.readPixels (dw.min.y, dw.max.y);
 }
 
-// WIP
 void
 PyPart::readDeepPixels(MultiPartInputFile& infile, const std::string& type, const ChannelList& channel_list,
                        const std::vector<size_t>& shape, const std::set<std::string>& rgba_channels,
                        const Box2i& dw, bool rgba)
 {
     DeepFrameBuffer frameBuffer;
 
+    size_t width  = dw.max.x - dw.min.x + 1;
+    size_t height = dw.max.y - dw.min.y + 1;
+
+    Array2D<unsigned int> sampleCount (height, width);
+
+    frameBuffer.insertSampleCountSlice (Slice (
+                                            UINT,
+                                            (char*) (&sampleCount[0][0] - dw.min.x - dw.min.y * width),
+                                            sizeof (unsigned int) * 1,       // xStride
+                                            sizeof (unsigned int) * width)); // yStride
+
     for (auto c = channel_list.begin(); c != channel_list.end(); c++)
     {
         std::string py_channel_name = c.name();
+#if XXX
```

Comment:
is this saved for later, or currently not working?

Reply:
Sorry, the deep stuff is WIP, not fully working yet. It properly reads the data, but I'm still figuring out how to pack it into numpy arrays.